datadog/dogshell/monitor.py

```diff
@@ -7,10 +7,9 @@
 
 # datadog
 from datadog import api
-from datadog.dogshell.common import report_errors, report_warnings
+from datadog.dogshell.common import report_errors, report_warnings, print_err
 import warnings
```

Comment:
Let's remove this import which is now unused and add back the blank line that's making flake8 tests

Reply:
updated